bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/thing/plan.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="evcc"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<thing-type id="plan" listed="false">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="server"/>
+		</supported-bridge-type-refs>
+		<label>Plan</label>
+		<description>A plan configured for your vehicle in evcc</description>
```

Comment:
Can you add equipment tag ?

Reply:
Where?

Reply:
Line directly after the description tag.
For reference: https://www.openhab.org/docs/developer/bindings/thing-xml.html#thing-semantic-equipment-tags